plugins/com.gwtplugins.gdt.eclipse.suite/project_templates/client_server_shared/client/.classpath

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="com.gwtplugins.gwt.eclipse.core.GWT_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
```

Comment:
What happens if JavaSE-11 is not configured?

Reply:
I think what Luca is asking is if we can be sure that Java 11 will be installed if 17 is the baseline - 11 may not be installed at all.

Does `JavaSE-11` implicitly include all future releases? Java 21 for example has deprecated Java 8 source/target/release, it is reasonable that a future release might drop 11.

Reply:
JavaSE-11 is an Execution Environment. It does not mean an exact Java Version. For example in my current Eclipse i run these Applications with Java 17.
At the moment all versions > 11 can be used to run applications with the JavaSE-11 Execution Environment.
I cannot make sure, that Java 30 or Eclipse 2026 will still be compatible, but for now this should be OK.
Anyway, it is a template file. Users could change the required Execution Environment as they need it.